src/main/java/net/atos/zac/aanvraag/ProductAanvraagService.java

```diff
@@ -68,6 +74,15 @@ public void createZaak(final URI productAanvraagUrl) {
 
         pairProductAanvraagWithZaak(productAanvraagUrl, zaak.getUrl());
         pairAanvraagPDFWithZaak(productAanvraag, zaak.getUrl());
+        addInitiator(productAanvraag.getBsn(), zaak.getUrl(), zaak.getZaaktype());
```

Comment:
Kun je hier de bsn al controleren op != null (of misschien isBlank(), wil je wel een initator toevoegen als het bsn leeg is?). Want deze methode doet niets als het bsn null blijkt te zijn, fijner als je dat gegeven hier al kunt zien.

Reply:
De logica in addInitiator gaat complexer worden als een initiator straks ook een bedrijf is.